src/mango_idx_text.erl

```diff
@@ -261,4 +266,9 @@ indexable_fields(Fields, {op_null, {_, _}}) ->
     Fields;
 
 indexable_fields(Fields, {op_default, _}) ->
-    [<<"$default">> | Fields].
+    [<<"$default">> | Fields];
+
+%% Some lingering term that made it through so rather than throwing
+%% an error, we just return fields.
+indexable_fields(Fields, _) ->
+    Fields.
```

Comment:
This also seems ungood. Not allowing random non-conforming normalized selectors through here seems like not a good idea if we don't have to.